src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/UserManagementServicePermissionTests.cs

```diff
@@ -0,0 +1,158 @@
+using System;
+using GovUk.Education.ExploreEducationStatistics.Admin.Areas.Identity.Data;
+using GovUk.Education.ExploreEducationStatistics.Admin.Areas.Identity.Data.Models;
+using GovUk.Education.ExploreEducationStatistics.Admin.Models;
+using GovUk.Education.ExploreEducationStatistics.Admin.Security;
+using GovUk.Education.ExploreEducationStatistics.Admin.Services;
+using GovUk.Education.ExploreEducationStatistics.Common.Services.Interfaces.Security;
+using GovUk.Education.ExploreEducationStatistics.Common.Utils;
+using GovUk.Education.ExploreEducationStatistics.Content.Model.Database;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.Configuration;
+using Moq;
+using Xunit;
+using static GovUk.Education.ExploreEducationStatistics.Admin.Tests.Services.PermissionTestUtil;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Tests.Services
+{
+    public class UserManagementServicePermissionTest
+    {
+        [Fact]
+        public void ListPendingInvitesAsync()
+        {
+            PolicyCheckBuilder()
+                .ExpectCheck(SecurityPolicies.CanManageUsersOnSystem)
+                .AssertSuccess(
+                    async userService =>
+                    {
+                        await using (var userAndRolesDbContext =
+                            DbUtils.InMemoryUserAndRolesDbContext(Guid.NewGuid().ToString()))
+                        {
+                            var userManagementService = BuildUserManagementService(
+                                userAndRolesDbContext,
+                                userService.Object);
+                            return await userManagementService.ListPendingInvitesAsync();
+                        }
+                    });
+        }
+
+        [Fact]
+        public void InviteUserAsync()
+        {
+            PolicyCheckBuilder()
+                .ExpectCheck(SecurityPolicies.CanManageUsersOnSystem)
+                .AssertSuccess(
+                    async userService =>
+                    {
+                        var userEmail = "test@test.com";
+                        var inviteCreatedByUser = "Test User";
+                        var roleId = Guid.NewGuid().ToString();
+                        var user = new ApplicationUser
+                        {
+                            FirstName = "TestFirstName",
+                            LastName = "TestLastName",
+                        };
+                        var role = new IdentityRole
+                        {
+                            Id = roleId
```

Comment:
You may as well use `Id = Guid.NewGuid().ToString();` here and then refer to it by `role.Id`